python-selenium/src/bandcamp/app/tui.py

```diff
@@ -0,0 +1,74 @@
+from bandcamp.app.player import Player
+
+COLUMN_WIDTH = CW = 30
+MAX_TRACKS = 100  # Allows to load more tracks once.
+
+
+def interact():
+    """Control the player through user interactions."""
+    with Player() as player:
+        while True:
+            print(
+                "\nType: play [<track number>] | pause | tracks | more | exit"
+            )
+            match input("> ").strip().lower().split():
+                case ["play"]:
+                    play(player)
+                case ["play", track]:
+                    try:
+                        track_number = int(track)
+                        play(player, track_number)
+                    except ValueError:
+                        print("Please provide a valid track number.")
+                case ["pause"]:
+                    pause(player)
+                case ["tracks"]:
+                    display_tracks(player)
+                case ["more"] if len(
+                    player.tracklist.available_tracks
+                ) >= MAX_TRACKS:
+                    print(
+                        "Can't load more tracks. Pick one from the track list."
+                    )
+                case ["more"]:
+                    player.tracklist.load_more()
+                    display_tracks(player)
+                case ["exit"]:
+                    print("Exiting the player...")
+                    break
+                case _:
+                    print("Unknown command. Try again.")
+
+
+def play(player, track_number=None):
+    """Play a track and shows info about the track."""
+    player.play(track_number)
+    print(player._current_track._get_track_info())
```

Comment:
@martin-martin It would be nice to include some rudimentary error handling. For example, when I request to play a track outside the valid range, the script crashes abruptly.